plugins/tiddlywiki/dynaview/dynaview.js

```diff
@@ -139,6 +139,36 @@ function checkVisibility() {
 	$tw.utils.each(elements,function(element) {
 		// Bail if we've already triggered this element
 		if(element.getAttribute("data-dynaview-has-triggered") === "true") {
+			if(element.getAttribute("data-dynaview-unset-tiddler") !== undefined && element.getAttribute("data-dynaview-unset-value") !== undefined) {
+				// Check if the element is visible
+				var elementRect = element.getBoundingClientRect(),
+					viewportWidth = window.innerWidth || document.documentElement.clientWidth,
+					viewportHeight = window.innerHeight || document.documentElement.clientHeight,
+					viewportRect = {
+						left: 0,
+						right: viewportWidth,
+						top: 0,
+						bottom: viewportHeight
+					};
+				if(element.classList.contains("tc-dynaview-expand-viewport")) {
+					viewportRect.left -= viewportWidth;
+					viewportRect.right += viewportWidth;
+					viewportRect.top -= viewportHeight;
+					viewportRect.bottom += viewportHeight;
+				}
+				if(elementRect.left > viewportRect.right || 
+					elementRect.right < viewportRect.left || 
+					elementRect.top > viewportRect.bottom ||
+					elementRect.bottom < viewportRect.top) {
+					// Set the tiddler value
+					var tiddler = element.getAttribute("data-dynaview-unset-tiddler"),
+						value = element.getAttribute("data-dynaview-unset-value") || "";
+					if(tiddler && $tw.wiki.getTiddlerText(tiddler) !== value) {
+						$tw.wiki.addTiddler(new $tw.Tiddler({title: tiddler, text: value}));
+					}
+					element.setAttribute("data-dynaview-has-triggered",false);
```

Comment:
For consistency with the test in line 141 perhaps this should be set to the string `"false"`.

Reply:
@Jermolene - I'm reading "false" here - do I have to be worried about dyslexia? ;)